src/localization/localization.ts

```diff
@@ -263,23 +267,11 @@ function repository(type: RepositoryType): string {
     }
 }
 
-function dateSlot(slot: DateSlot) {
-    switch (slot) {
-        case DateSlot.Today:
-            return "Today";
-        case DateSlot.Yesterday:
-            return "Yesterday";
-        case DateSlot.LessThanAWeekAgo:
-            return "Less than a week ago";
-        case DateSlot.OverAWeekAgo:
-            return "Over a week ago";
-        case DateSlot.OverAMonthAgo:
-            return "Over a month ago";
-        case DateSlot.OverAYearAgo:
-            return "Over a year ago";
-        default:
-            throw new Error(`Unhandled date slot: ${slot}`);
-    }
+function getMonthName(month: number) {
+    return [
+        "January", "February", "March", "April", "May", "June",
+        "July", "August", "September", "October", "November", "December"
+    ][month];
```

Comment:
Let's make this array a constant up top 😄

Reply:
This is on purpose although it does not add much for this project. I prefer not to make localization strings constants because if we add a hypothetical feature to change localization while app is active, the constant strings will be continued to used and text will be shown in wrong languages.